it.js

```diff
@@ -0,0 +1,777 @@
+module.exports = `
+menu.btn.ranked Classificata
+menu.btn.shop Shop
+menu.btn.social Social
```

Comment:
Are we able to translate these?

Reply:
Shop was left like that in order to avoid confusion with: "windows.store.header Store" and "windows.store.market Marketplace".
Both: **Shop**, **Marketplace** and **Store** are translated as "**Negozio**" in italian.
If you wish we can translate them all.

Reply:
gira can i help for trasletion?

Reply:
"Mercatino" is more confusing than shop since it is used to indicate thrift shops.